index.js

```diff
@@ -147,6 +148,7 @@ client.once(Events.ClientReady, (client) => {
   })
   BipMessages.sync()
   ManageChannels.sync()
+  UserAddresses.sync()
```

Comment:
as `UserAddresses` references many `UserInscriptions` it should be created after with `UserInscriptions.sync().then(...`